mmdet/core/bbox/assigners/max_iou_assigner.py

```diff
@@ -23,6 +23,10 @@ class MaxIoUAssigner(BaseAssigner):
         min_pos_iou (float): Minimum iou for a bbox to be considered as a
             positive bbox. Positive samples can have smaller IoU than
             pos_iou_thr due to the 4th step (assign max IoU sample to each gt).
+            `min_pos_iou` is set to avoid assigning bboxes that have extremely
+            small iou with GT as positive samples. It brings about 0.3 mAP
+            improvements and more comparisons can be found in
+            https://github.com/open-mmlab/mmdetection/pull/7464.
```

Comment:
Here we can use sphinx syntax like 
```
`PR #7464 <https://github.com/open-mmlab/mmdetection/pull/7464>`_
```

Reply:
May check https://www.sphinx-doc.org/zh_CN/master/usage/restructuredtext/basics.html

Reply:
Done.